examples/medplum-patient-intake-demo/src/App.tsx

```diff
@@ -42,6 +47,7 @@ export function App(): JSX.Element | null {
             <Route path="/:resourceType/:id/*" element={<ResourcePage />} />
             <Route path="/:resourceType" element={<SearchPage />} />
             <Route path="/upload/:dataType" element={<UploadDataPage />} />
+            <Route path="/customize-form" element={<QuestionnaireCustomizationPage />} />
```

Comment:
In general, we try to avoid custom routes like this. Rather we should link to `/Questionnaire/:id/edit`

Reply:
To find the ID of the intake form, do the following: 
1. Use the `useSearchResources` hook to find all questionnaires with `name=Patient Intake Form`
2. Select the first return value as the intake form
3. Create a React Context for the app, that stores the intake questionnare
4. Remove all hardcoded ids for the intake form, and instead use this context